package.json

```diff
@@ -30,8 +30,8 @@
     "generate:feature": "hygen feature new",
     "generate:chain": "hygen chain new",
     "prepare": "husky install",
-    "test:changed-feature-branch": "jest --changedSince=origin/canary --coverage --passWithNoTests",
-    "lint:changed-feature-branch": "eslint --max-warnings 0 --no-error-on-unmatched-pattern $(git diff --name-only origin/canary | grep -E \"(.js$|.jsx$)\" || echo \"no js/jsx files changed\")",
+    "test:changed-feature-branch": "jest --changedSince=origin/arc-themes-release-version-1.19 --coverage --passWithNoTests",
```

Comment:
These changes are in another PR too - should we create a separate PR to address the issue/needs for this?

Reply:
Curious what the takeaways will be for the pr doing this for themes components. This is possibly the wrong place for a long term solution but needed this for getting past linting locally

Reply:
With the move to release based branching - and the speed it takes to lint file - do we need the overhead of trying to only lint the difference? Could we just run the `npm run lint` command on push?